stable/redis/templates/deployment.yaml

```diff
@@ -45,6 +45,24 @@ spec:
         volumeMounts:
         - name: redis-data
           mountPath: /bitnami/redis
+{{- if .Values.metrics.enabled }}
+      - name: metrics
+        image: "{{ .Values.metrics.image }}:{{ .Values.metrics.imageTag }}"
+        imagePullPolicy: {{ default "" .Values.metrics.imagePullPolicy | quote }}
```

Comment:
Avoid using the `default` function if you can - it makes it harder for end-users to determine what the value will be. In this case, set `metrics.imagePullPolicy` explicitly in `values.yaml`.